### ISSUE TITLE
Cutting and pasting a line produces an extra new line in Markdown mode

### ISSUE THREAD

**Issue by acrodrig** [HappenApps/Quiver#571]
Description within a Markdown cell: when I select a line (via `Shift+DownArrow`) and then cut (`Cmd+C`) and paste (`Cmd+V`) immediately Quiver will insert an extra new line. See video attached. 

I get the same behavior when pasting a line repeatedly which is annoying because I have to go back and remove the extra new lines.

![quiver](https://cloud.githubusercontent.com/assets/959336/14993230/b6bed08e-111e-11e6-91e6-fcff8ec5b92e.gif)

**Comment by olidevelop** [comment]
I agree, this behavior is annoying, no other app (that I know) does this. Please fix!
But I don't want to miss to say this app is really good! I'm looking forward the iOS version :-)))

**Comment by ylian** [comment]
Yep, this is annoying. Will fix.
